src/components/SingletonEditor.test.tsx

```diff
@@ -40,60 +39,97 @@ const queryClient = new QueryClient({
   },
 });
 
-describe(Editor.name, () => {
-  it("should show and hide EmailEditor when design is set and unset", async () => {
-    // Arrange
-    const appServices = defaultAppServices as AppServices;
+const emptyContent: Content = {
+  htmlContent: "",
+  type: "unlayer",
+  design: {
+    body: {
+      rows: [],
+    },
+  },
+};
+
+describe(`${SingletonEditorProvider.name}`, () => {
+  // Arrange
+  const appServices = defaultAppServices as AppServices;
+
+  const DemoComponent = ({ onSave }: { onSave?: () => void }) => {
+    const [initialContent, setInitialContent] = useState<Content | undefined>(
+      undefined
+    );
```

Comment:
Muy menor y no estoy seguro de que el compilador lo acepte. Pero, de ser así, sería más limpio:

```suggestion
    const [initialContent, setInitialContent] = useState<Content | undefined>();
```

Reply:
Si lo toma, es equivalente a colocar _**undefined**_